x/staking/types/errors.go

```diff
@@ -50,4 +50,5 @@ var (
 	ErrCommissionLTMinRate             = errors.Register(ModuleName, 40, "commission cannot be less than min rate")
 	ErrUnbondingNotFound               = errors.Register(ModuleName, 41, "unbonding operation not found")
 	ErrUnbondingOnHoldRefCountNegative = errors.Register(ModuleName, 42, "cannot un-hold unbonding operation that is not on hold")
+	ErrInvalidSigner                   = errors.Register(ModuleName, 43, "expected gov account as only signer for proposal message")
```

Comment:
This should be renamed. The authority can be any account and not only gov. Depends on the module config, so the error is confusing.